tests/apps/pubsub-subscriber/app.go

```diff
@@ -237,6 +264,18 @@ func extractMessage(body []byte) (string, error) {
 		return "", err
 	}
 
+	if m["data_base64"] != nil {
+		b, err := base64.StdEncoding.DecodeString(m["data_base64"].(string))
+		if err != nil {
+			log.Printf("Could not base64 decode, %s", err.Error())
+			return "", err
+		}
+
+		msg := string(b)
+		log.Printf("output='%s'\n", msg)
+		return msg, nil
+	}
```

Comment:
Sorry if I'm missing something, but are we encoding the raw payload as base64? I don't see duplicate code in the grpc side or in the publisher side so I'm confused as to where this is coming from.

Reply:
Great question. This problem only exists in HTTP, based on the CE spec in JSON. The gRPC subscriber does not have this problem because it is handled in the runtime: https://github.com/dapr/dapr/blob/09d0fd96a25aea4f96f780ddf5b61ae20ed537af/pkg/runtime/runtime.go#L1228